src/main/scala/com/resy/ResyBookingBot.scala

```diff
@@ -74,9 +74,9 @@ object ResyBookingBot extends App with StrictLogging {
     logger.info("Within booking window for venue - attempting immediate reservation")
     bookReservationWorkflow()
   } else {
-    val durationToSleep: FiniteDuration = config.minutesToBookingWindowStart(leadTime)
+    val durationToSleep: FiniteDuration = config.secondsToBookingWindowStart(leadTime) - config.wakeAdjustment
```

Comment:
Add a wakeAdjustment to allow the bot to start sniping a few seconds prior to the booking window opening to allow for clcok inconsistencies.